### PR TITLE
Remove persistence of grouping settings

### PR DESCRIPTION
**Pull request by mohammedhemaid**
The video after the removing:
https://photos.app.goo.gl/JJxTnYz54jWKJPCg9
sorry for making the first letter of the commit in lowercase

### CODE REVIEW

File: app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingMakingFragment.java [hunk 1]
```diff
@@ -48,7 +48,6 @@ public static GroupingMakingFragment getInstance(String listName) {
     private String listName;
     private DataSource dataSource;
```
Comment (Gear61) [inline app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingMakingFragment.java]: We can remove DataSource too right?

**Review by Gear61 — CHANGES_REQUESTED** [review]

**Comment by Gear61** [comment]
Commit casing isn't a huge deal, don't worry about it. :)

File: app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingMakingFragment.java (hunk 1 above)
Comment (mohammedhemaid) [reply app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingMakingFragment.java]: @Gear61  you mean to make it as a local variable?

Comment (mohammedhemaid) [reply app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingMakingFragment.java]: or making an object of it like this:
image

**Review by Gear61 — APPROVED** [review]
Awesome, looks good to me! Merging.

**Pull request merged**